GCC has an extension to evaluate contant expressions at 
compile time even though they are not compile time constants according
to the C standard. This problem has been reported to the NTP community
as NTP bug 3584: http://bugs.ntp.org/show_bug.cgi?id=3584
This patch may be removed once that bug is fixed.

*** ntpd/ntp_control.c
--- ntpd/ntp_control.c
*************** ctl_putsys(
*** 1916,1922 ****
  # ifdef STA_NANO
  		1.0e-6; /* nsec to msec */
  # else
! 		to_ms_usec;
  # endif
  
  	/*
--- 1916,1922 ----
  # ifdef STA_NANO
  		1.0e-6; /* nsec to msec */
  # else
! 		1.0e-3; /* usec to msec */
  # endif
  
  	/*
